src/components/reviewMatches/ReviewLink.tsx

```diff
@@ -190,94 +190,127 @@ const ReviewLink = () => {
     return <Loading />
   }
 
-  if (isError) {
+  if (isError && error) {
     return <ApiErrorMessage error={error} />
   }
```

Comment:
If error is empty + isError is true, it will not display the error message. We should add a default value for the prop "error" in ApiErrorMessage component.